CPP17.md

```diff
@@ -178,6 +178,17 @@ const auto [ x, y ] = origin();
 x; // == 0
 y; // == 0
 ```
+```c++
+std::unordered_map<std::string, int> mapping {
+  {"a", 1},
+  {"b", 2},
+  {"c", 3},
+};
+
+for (const auto [key, value] : mapping) {
```

Comment:
I would change this to:
```cpp
// Destructure by reference.
for (const auto& [key, value] : mapping) {
```
Generally you don't want to copy unless you have to.

Reply:
Made key, value const auto&